src/components/Radio/RadioGroup.test.jsx

```diff
@@ -12,123 +15,124 @@ async function axeTest(renderResult) {
   expect(results).toHaveNoViolations();
 }
 
-const DefaultRadioArray = [
-  {
-    label: "Radio 1",
-    value: "Radio 1",
-  },
-  {
-    label: "Radio 2",
-    value: "Radio 2",
-  },
-  {
-    label: "Radio 3",
-    value: "Radio 3",
-  },
-  {
-    label: "Radio 4",
-    value: "Radio 4",
-    disabled: true,
-  },
-  {
-    label: "Radio 5",
-    value: "Radio 5",
-    disabled: true,
-    tooltip: "Tooltip for Radio",
-    position: "down",
-  },
-];
-
-const DefaultProps = {
-  radios: DefaultRadioArray,
-  groupName: "Radio Group",
-  checked: "Radio 1",
-};
+const { RadioGroupExample, InlineRadioGroupExample } =
+  composeStories(RadioGroupStories);
 
 describe("RadioGroup", () => {
   let renderResult;
 
   beforeEach(() => {
-    vi.spyOn(console, "warn").mockImplementation(() => null);
-
-    renderResult = render(<RadioGroup {...DefaultProps} />);
+    renderResult = render(
```

Comment:
Kent C Dodds (creator of React Testing Library) [recommends](https://kentcdodds.com/blog/common-mistakes-with-react-testing-library#not-using-screen) using _just_ `render` and then using `screen.*`.

eg:
```suggestion
   render(
```

and then further down (line 34):
```
const rootElement = screen.getByRole("radiogroup");
```

And, with Vite, you'd also want to add a siblig to the `beforeEach` that is:
```
    afterEach(() => {
      cleanup();
    });
```

You can see an example of where I've used this strategy in `src/components/Tab/Tabs.test.jsx` [line 93-155](https://github.com/avaya-dux/neo-react-library/blob/main/src/components/Tab/Tabs.test.jsx) ("`describe("Closable Tab"`")

Reply:
You don't have to do this, most of our tests are the way that you have it written here. But from my reading, using `screen.*` is the safer/better way. And if you are doing the `render` in a `beforeEach`, you should also do a `afterEach` cleanup for safety.

Reply:
Interesting; I was not aware of this pattern/recommendation. You are right it does seem like the recommended way. I have made the changes as suggested

Reply:
Yeah, the majority of our tests don't follow best practices. I posted that article [back in February](https://designavaya.slack.com/archives/C02BH0B4NUA/p1645812324780899) in git-good, but it's _pretty_ long so I don't think that many people read it. I'll bring this up in our Coding Discussion today. So that we can all get on the same page going forward.